ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailActions.java

```diff
@@ -0,0 +1,8 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+public class IntegrationDetailActions {
```

Comment:
Enum maybe?

Reply:
I'm not sure if we would really benefit from it.